content/S2016/S16-event-wics-csc.md

```diff
@@ -4,10 +4,16 @@ Category: Events
 Tags: social
 Slug: wics-csc-s16
 Author: Clare Ng
-Summary: We're having a bonfire with the CSC! There will be ice cream and marshmellows.
+Summary: We're having a bonfire with the CSC! There will be ice cream and 
+marshmellows.
```

Comment:
not the summary though ^_^ because this is like meta data and has to be all one on line